### ISSUE TITLE
WKWebView doesn't rotate contents correctly on iOS 10

### ISSUE THREAD

**Issue by TsvetinaIvanova** [Telerik-Verified-Plugins/WKWebView#260]
When the WKWebView is used on iOS 10, rotating the screen doesn't trigger a resize of the HTML contents: http://screencast.com/t/EJkj7CVn.

The same project works as expected using the default WebView. You can get project from here:
https://drive.google.com/file/d/0B38gHd-jMfVkUVVzQWF1SnlBUEE/view?usp=sharing

**Comment by EddyVerbruggen** [comment]
I don't think it's an iOS 10 issue as [this has been reported before (for PhoneGap Build mostly)](https://github.com/Telerik-Verified-Plugins/WKWebView/search?q=landscape&type=Issues&utf8=%E2%9C%93)

I am not able to reproduce this with a Cordova CLI project, but I am able to reproduce it on TP. I've compared many differences between those two projects but I can't find the thing that's causing the difference..

I've stripped all the plugins from the app you linked to except this one, also built it for debugging and release, and copied the webfolder to my local project as well. I can't break my CLI project and I can't repair the TP project.

Perhaps it's an option for the customer to limit the rotation to portrait on iOS and use the [screenorientation plugin](https://github.com/EddyVerbruggen/cordova-plugin-screenorientation-for-wkwebview) to force orientation changes when/if they make sense.
